pkg/cluster/controllers/database/utils.go

```diff
@@ -17,12 +16,12 @@ const (
 
 func (p *PostgreSQLController) GetDatabases() map[string]string {
 	databases := map[string]string{
-		CoreDatabase: harbormetav1.CoreDatabase,
+		CoreDatabase: DefaultDatabaseName,
 	}
 
 	if p.HarborCluster.Spec.Notary != nil {
-		databases[NotaryServerDatabase] = harbormetav1.NotaryServerDatabase
-		databases[NotarySignerDatabase] = harbormetav1.NotarySignerDatabase
+		databases[NotaryServerDatabase] = DefaultDatabaseName
+		databases[NotarySignerDatabase] = DefaultDatabaseName
```

Comment:
Check the code:

```go
func GetUsers() map[string]api.UserFlags {
	return map[string]api.UserFlags{
		"zalando": {
			"superuser",
			"createdb",
		},
		"foo_user": {},
	}
}
```

you did not create `DefaultDatabaseName` here, the user you created is `zalando`.